src/plugins/controls/public/control_group/embeddable/control_group_container.tsx

```diff
@@ -100,6 +162,10 @@ export class ControlGroupContainer extends Container<
     });
   }
 
+  public getPanelCount = () => {
```

Comment:
I wonder if it's worth moving this into the generic container.ts - maybe not as part of this PR because it would trigger an app services review, but in a followup.

Reply:
Yes, yes! I definitely thought about putting it there, but decided to keep it local to controls for now. Would be awesome to pull it out for sure 💪